ext/DimensionalDataDiskArraysExt.jl

```diff
@@ -0,0 +1,18 @@
+"""
+    DimensionalDataDiskArraysExt
+
+Extend some methods of DiskArrays (`cache`, etc) to work on the base data of any DimArray.  
+"""
+module DimensionalDataDiskArraysExt
+
+using DimensionalData
+import DimensionalData: AbstractBasicDimArray
+import DiskArrays
+
+DiskArrays.cache(x::Union{AbstractDimStack,AbstractArray}; kw...) = modify(A -> DiskArrays.cache(A); kw...))
```

Comment:
```suggestion
DiskArrays.cache(x::Union{AbstractDimStack,AbstractDimArray}; kw...) = modify(A -> DiskArrays.cache(A; kw...))
```

Sorry typed it wrong on my phone